ckanext/dms/helpers.py

```diff
@@ -1,5 +1,5 @@
-import ckan.logic as logic
 import ckan.model as model
+import ckan.logic as logic
```

Comment:
Try you're best to avoid importing core CKAN.  It should in theory be possible to do everything using the ckan plugins toolkit. 
```suggestion
from ckan.plugins import toolkit
```